lib/contestant.rb

```diff
@@ -0,0 +1,34 @@
+class Contestant
+  attr_reader :first_name,
+              :last_name,
+              :age,
+              :state_of_residence,
+              :spending_money,
+              :game_interests
+
+  def initialize(contestant_info)
+    @first_name = contestant_info[:first_name]
+    @last_name = contestant_info[:last_name]
+    @age = contestant_info[:age]
+    @state_of_residence = contestant_info[:state_of_residence]
+    @spending_money = contestant_info[:spending_money]
+    @game_interests = []
+  end
+
+  def full_name
+    @first_name + " " + @last_name
+  end
+
+  def out_of_state?
+    if @state_of_residence == "CO"
```

Comment:
You could simplify this to `!(@state_of_residence == "CO")` or `@state_of_residence != "CO"`